app/languages/en/user/inventoryOverview.json

```diff
@@ -13,5 +13,6 @@
   "inventory_overview_loc_right_text": "{{item.treeCount}} trees",
   "inventory_overview_loc_export_json": "Export GeoJson",
   "inventory_overview_loc_next_tree": "Next Tree",
-  "inventory_overview_loc_save": "Save"
+  "inventory_overview_loc_save": "Save",
+  "inventory_overview_date": "{{date, ll}}"
```

Comment:
I don't think this needs to be translated

Reply:
Maybe it's just for date formatting if we do not have any other module for that in this project?

Reply:
from i18next docs, date can also be translated. maybe @norbertschuler can confirm if it is necessary.

Reply:
I have not yet used i18n for localization purposes. Would be nice it it works. I will test this PR as soon as I have some time left.

Reply:
This way the date is not translated ("Oct" in German would have to be written as "Okt"):

![Simulator Screen Shot - iPhone 11 - 2020-11-09 at 18 10 41](https://user-images.githubusercontent.com/1532418/98573283-42e59a80-22e9-11eb-919b-342b2f4f928a.png)

Reply:
I have now added a commit doing the i18n + Moment.js date localization correctly.